### PR TITLE
3589 get permissions for thousands entities

### LINKED ISSUE #3589: Updating permissions on big amount of entities may cause nginx error
By design, the permissions modal issues a GET request to the server to get the merged permissions for the selected entities.
The endpoint, being a GET method, receives the id's of the entities on the URL query. That means that selecting, let's say, 1000 entities will result in a query including 1000 id's, which causes a `URL too long` error on nginx.

### PR DESCRIPTION
**Pull request by mfacar**
fixes #3589

PR checklist:
- [ ] Update READ.me ?
- [ ] Update API documentation ?

QA checklist:
- [ ] Smoke test the functionality described in the issue
- [ ] Test for side effects
- [ ] UI responsiveness
- [ ] Cross browser testing
- [ ] Code review

### CODE REVIEW

**Review by fnocetti — COMMENTED** [review]
@mfacar the CR LGTM. I'll proceed to the manual testing.

**Comment by fnocetti** [comment]
> @mfacar the CR LGTM. I'll proceed to the manual testing.

The manual testing went just fine. I'll wait for your comments to merge.

File: app/api/permissions/routes.ts [hunk 1]
```diff
@@ -26,14 +26,18 @@ export const permissionRoutes = (app: Application) => {
...
-      const permissions = await entitiesPermissions.get(req.query.ids);
-      res.json(permissions);
-    } catch (err) {
-      next(err);
+  app.put(
+    '/api/entities/permissions/',
+    needsAuthorization(['admin', 'editor', 'collaborator']),
+    async (req, res, next) => {
+      try {
+        const permissions = await entitiesPermissions.get(req.body);
```
[the remark this answers is not in the document]
Comment (mfacar) [reply app/api/permissions/routes.ts:34]: You're right, I'll change it 👍

**Review by fnocetti — APPROVED** [review]
Merging

**Pull request merged**